src/cli/commands/install.js

```diff
@@ -165,13 +166,13 @@ export class Install {
   constructor(flags: Object, config: Config, reporter: Reporter, lockfile: Lockfile) {
     this.rootManifestRegistries = [];
     this.rootPatternsToOrigin = map();
-    this.resolutions = map();
     this.lockfile = lockfile;
     this.reporter = reporter;
     this.config = config;
     this.flags = normalizeFlags(config, flags);
-
-    this.resolver = new PackageResolver(config, lockfile);
+    this.resolutions = map();
+    this._resolutions = new Resolutions(config);
```

Comment:
hm so `this.resolutions` was what was being previously used for flat mode - I didn't necessarily want to mess with it but the rfc said the plan was to eventually have flat mode replaced by resolutions so I kind of left both as is. How would you rename it?